pygmt/helpers/testing.py

```diff
@@ -161,6 +161,7 @@ def download_test_data():
         "@earth_relief_10m_g",
         "@earth_relief_05m_p",
         "@earth_relief_05m_g",
+        "@earth_gebco_01d_g",
```

Comment:
Suggest to move this to L156 to sort alphabetically.

Reply:
Moved.